homeassistant/components/ring/light.py

```diff
@@ -0,0 +1,97 @@
+"""This component provides HA switch support for Ring Door Bell/Chimes."""
+import logging
+from datetime import datetime, timedelta
+from homeassistant.components.light import Light
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.core import callback
+
+from . import DATA_RING_STICKUP_CAMS, SIGNAL_UPDATE_RING
+
+_LOGGER = logging.getLogger(__name__)
+
+"""
+It takes a few seconds for the API to correctly return an update indicating
+that the changes have been made. Once we request a change (i.e. a light
+being turned on) we simply wait for this time delta before we allow
+updates to take place.
+"""
+SKIP_UPDATES_DELAY = timedelta(seconds=5)
+
+ON_STATE = "on"
+OFF_STATE = "off"
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Create the lights for the Ring devices."""
+    cameras = hass.data[DATA_RING_STICKUP_CAMS]
+    lights = []
+
+    for device in cameras:
+        if device.has_capability("light"):
+            lights.append(RingLight(device))
+
+    add_entities(lights, True)
+
+
+class RingLight(Light):
+    """Creates a switch to turn the ring cameras light on and off."""
+
+    def __init__(self, device):
+        """Initialize the light."""
+        self._device = device
+        self._unique_id = "{}-light".format(self._device.id)
```

Comment:
Remove light suffix. The core is aware of the domain and platform when checking uniqueness.

Reply:
Should I do this same for the switch platform?

Reply:
Yes please.

Reply:
hmm, there will be more in future. I'm going to leave it.